docs/components/GettingStartedView.jsx

```diff
@@ -1,14 +1,81 @@
 import React from "react";
 
+import CodeSample from "./Example/CodeSample";
 import View from "./View";
-
+import {Grid} from "src";
 
 export default function GettingStartedView() {
   const {cssClass} = GettingStartedView;
+  const {Col, Row} = Grid;
 
   return (
     <View className={cssClass.CONTAINER} title="Getting Started">
-      TODO
+      <Grid>
+        <Row grow className="items--stretch margin--bottom--xl">
+          <Col span={6} className="self--start padding--right--l">
+            <h2>Install</h2>
+            <p>Install the NPM package and save it to your project using</p>
+            <CodeSample>npm install --save clever-components</CodeSample>
+            <p>Note: This library uses ES6 features not supported in Internet Explorer such as <a className="Button Button--linkPlain Button--regular" href="https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith">String.startsWith</a>. To support use of this library in IE, include <code className="text--small">babel-polyfill</code> in the entrypoint to your application. See the <a className="Button Button--linkPlain Button--regular" href="https://babeljs.io/docs/usage/polyfill/">docs</a>.</p>
```

Comment:
similar comment about wrapping long lines here. long URLs are an exception, since they don't break easily.
should also indent nested elements for readability
```jsx
<p>
  Note: ...
  ...such as
  {" "} // Explicit spaces are needed for newlines right before an opening tag
  <a 
    className=...
    href="..." // props can be wrapped if the opening tag is too long
  >
    String.startsWith
  </a>.
  To support...
</p>
```